packages/create/create.js

```diff
@@ -837,7 +836,7 @@ async function main() {
 			nargs: 1, describe: "Path to Factorio installation [standalone/host]", type: "string",
 		})
 		.option("remote-npm", {
-			nargs: 0, description: "Allow remote usage of npm [standalone/controller/host]", type: "boolean",
+			nargs: 0, description: "Allow remote updates of npm [standalone/controller/host]", type: "boolean",
```

Comment:
"Allow remoter updates from npm" - we aren't updating the users npm install

Reply:
Missed this, updated to "via" to match the words used in the prompt.